adapters/repos/db/inverted/new_prop_length_tracker.go

```diff
@@ -76,19 +89,28 @@ func NewJsonPropertyLengthTracker(path string) (*JsonPropertyLengthTracker, erro
 		}
 		return nil, err
 	}
-	t.path = path
 
-	var data PropLenData
-	if err := json.Unmarshal(bytes, &data); err != nil {
-		if bytes[0] != '{' {
+	if len(bytes) == 0 {
+		// Something created the file but left it empty?  Log it and return an empty tracker
+		log.Printf("Property length tracker file %s was empty", path)
```

Comment:
I'm curious: Is this just for debugging, or will this also stay in the final fix? If so, I think this might make the current chaos pipeline which discovered this ineffective because it would no longer panic. We wouldn't see the logs if the bug reappeared, so we'd have a green pipeline and wouldn't know.